public/app/features/comments/CommentView.tsx

```diff
@@ -29,7 +29,7 @@ export const CommentView = ({ comments, packetCounter, addComment }: Props) => {
   }, [packetCounter]);
 
   const onUpdateComment = (event: FormEvent<HTMLTextAreaElement>) => {
-    const element = event.target as HTMLInputElement;
+    const element = event.currentTarget;
```

Comment:
Hm I've been running into this type assertion a lot, good to know that `currentTarget` is typed correctly!

Reply:
think https://stackoverflow.com/a/35597485 summarises it nicely

`target` is typed correctly i guess, it just has no way of knowing a more restricted type since it could come from basically anywhere due to event bubbling 😬

suspect in most cases (like this one) we only care about the element the event listener is attached to because we want to grab the value of the input